### DISCUSSION TITLE
Atmo Aura Api key

### DISCUSSION THREAD

**Discussion by farfromrefug** [breezy-weather/breezy-weather#1232] [Q&A]
Hi,
Where and how can we get an Atmo Aura API key?
I tried to create an account on http://api.atmo-aura.fr which gave me an API key.
But when used with `https://api.atmo-aura.fr/air2go/v3/point` it gives me the error : `Vous n'êtes pas authorisé à accéder à cette ressource` which leads me to believe my api key does not work with that air2go api.

**Comment by papjul** [comment]
The bundled key works on my side, did you encounter any issue that lead you to register your own key?

**Reply by farfromrefug** [reply]
@papjul no i dont. I just would prefer to use my own. Plus i am working on integrating atmo in another weather app. So i need to use my own key. Wont use breezy's one.
